testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -151,6 +151,19 @@ def test_toggle_atom_nomatch_mismatch_atoms(self, universe, reference):
         with pytest.raises(SelectionError):
             align.alignto(u, ref, select='all', match_atoms=False)
 
+    def test_no_atom_masses(self, universe):
+        #if no masses are present
+        u = mda.Universe.empty(6, 2, atom_resindex=[0, 0, 0, 1, 1, 1], trajectory=True)
+        with pytest.warns(SelectionWarning):
+            align.get_matching_atoms(u.atoms, u.atoms)
+    def test_one_atom_has_masses(self, universe):
```

Comment:
Please add a blank line between these functions. The name of this test is also not quite accurate; you are testing when one universe has masses, not when only one atom does.